fdbclient/FDBTypes.h

```diff
@@ -703,6 +703,8 @@ struct KeyValueStoreType {
 	// Only add new ones just before END.
 	// SS storeType is END before the storageServerInterface is initialized.
 	enum StoreType { SSD_BTREE_V1, MEMORY, SSD_BTREE_V2, SSD_REDWOOD_V1, MEMORY_RADIXTREE, SSD_ROCKSDB_V1, END };
+	constexpr static const StoreType ALL[] = { SSD_BTREE_V1,   MEMORY,           SSD_BTREE_V2,
```

Comment:
nit clang-format

Reply:
Unfortunately this is exactly what clang-format suggested.But anyways it not need anymore.